litecli/sqlcompleter.py

```diff
@@ -397,7 +397,7 @@ def find_matches(
         start_only: bool = False,
         fuzzy: bool = True,
         casing: str | None = None,
-        punctuations: str = "most_punctuations",
+        punctuations: LAST_WORD_INCLUDE_TYPE = "most_punctuations",
```

Comment:
Since the puncutations value is literal and here it was declared as str, ty complains about it. hence use the same type here.